app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @all_posts = @user.posts.includes(:comments)
+  end
```

Comment:
[OPTIONAL] I don't think any of this code will yield what you need for your post views. This controller is not meant to fetch posts for a user. Rather it is meant to fetch all posts

Reply:
resolved